skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BtReceiverActivity.java

```diff
@@ -93,6 +95,13 @@ protected void onCreate(Bundle savedInstanceState) {
         setTitle(getString(R.string.connect_bluetooth_title));
         setSupportActionBar(toolbar);
 
+        progressDialog = new ProgressDialog(this);
```

Comment:
Why creating a object in `onCreate`? It should be created whenever and wherever you want to show the dialog.

Reply:
resolved